### DISCUSSION TITLE
No more free 2 hours?

### DISCUSSION THREAD

**Discussion by Jokbal88** [kingparks/cursor-vip#44] [Q&A] [answered]
I have been using it and saw that there is no more free 2 hours option, is there any chance of it coming back? Been a life saver at my school projects, hope it comes back🙏🏼

**Comment by kingparks** [comment, marked as answer]
I'm glad he was able to help you. Currently, sharing a Pro account with 8 devices can sometimes be restricted, which increases our operating costs. Therefore, we have temporarily closed the free two-hour entrance. We will open it when we have new funding support!

**Reply by Jokbal88** [reply]
Thank you so much sir, I hope it works again since im working on a project, Cursor gets the most of it unlike ChatGPT 🙏🏼

**Comment by kingparks** [comment]
Give me your device code and tell me what you're going to accomplish with it, and I'll try to make a special case for you

**Reply by Jokbal88** [reply]
Thank you so much sir! Here is my DeviceID:137bd1f82d7743e5bea021ba8b1536c9, Im currently working on a piano application, i got assigned on a task to make them work on a websocket and other functionalities and your application really helped me alot!

**Reply by zmike808** [reply]
Is there a discount link to buy cursor for a year that you are offering? I was using cursor for developing a php project that I needed severe help with and the AI chat was extremely useful. Please let me know if we can talk privately @kingparks
